org-static-blog.el

```diff
@@ -302,10 +302,81 @@ Preamble and Postamble are excluded, too."
        (search-backward "</div>")
        (point)))))
 
-(defun org-static-blog-get-url (post-filename)
-  "Generate a URL to the published POST-FILENAME."
-  (file-name-nondirectory
-   (org-static-blog-matching-publish-filename post-filename)))
+(defun org-static-blog-get-absolute-url (relative-url)
+  "Returns absolute URL based on the RELATIVE-URL passed to the function.
+
+For example, when `org-static-blog-publish-url` is set to 'https://example.com/'
+and `relative-url` is passed as 'archive.html' then the function
+will return 'https://example.com/archive.html'."
+  (concat org-static-blog-publish-url relative-url))
+
+(defun org-static-blog-get-post-url (post-filename)
+  "Returns absolute URL to the published POST-FILENAME.
+
+This function concatenates publish URL and generated custom filepath to the
+published HTML version of the post."
```

Comment:
Could you add an explicit example here, like in `org-static-blog-get-absolute-url`?